### PR TITLE
add sound

### PR DESCRIPTION
**Pull request by psugihara**
This adds sound using tracks from one of the songs and muting/unmuting with probability.

I ended up using tone.js because I liked what I could do with filters. If we want to save file size, I can swap to howler. We'd lose the filter/reverb/compressor stuff but could still have stereo panning. I think I could make it still sound cool. This would probably save ~250k.

Another possibility is bouncing songs to 1 track (right now it's like 55kb per track, 5ish tracks per song). I think that would be much less cool though because I think the probabilistic track muting/unmuting is the most important thing for making it kind of interesting over long listening sessions.

### CODE REVIEW

**Comment by psugihara** [comment]
On the other hand, if we want more file size I can add more songs! :)

**Comment by okwme** [comment]
I can't hear the random muting / unmuting, maybe you have to memorize the original first?

**Comment by psugihara** [comment]
cool, will do 〰️ 〰️ 〰️ 〰️

File: src/sound.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      // type Track: [sample, probability, introProbability?]
+      [whistle_8_T7, 1, 0],
+      [whistle_4_T3, 0.9, 1],
+      [whistle_7_T6, 0.8, 1],
+      [whistle_12_T11, 0.7, 0],
+    ], [
+      [whistle_8_T7_B, 1, 0],
+      [whistle_4_T3, 0.9, 1],
+      [whistle_7_T6, 0.8, 1],
+      [whistle_12_T11, 0.7, 0],
```
Comment (psugihara) [inline src/sound.js:22]: It takes tracks in and out according to these probabilities. The melodies `whistle_8_T7` do not play at all during the intro then play 100% of the time.

**Comment by okwme** [comment]
> hmm actually something's breaking my webpack hot-reload in this pr, changing approval til i figure it out

i think it has something to do with the sound files, are you seeing an infinite loop reload when you make a change to `anybody.js` while `yarn dev` is running?

**Comment by psugihara** [comment]
ah yes it is fucked

**Comment by psugihara** [comment]
> hmm actually something's breaking my webpack hot-reload in this pr, changing approval til i figure it out

I see this in the main branch actually

**Comment by okwme** [comment]
> > hmm actually something's breaking my webpack hot-reload in this pr, changing approval til i figure it out
> 
> I see this in the main branch actually. Do you?

no i go back to main and it works again.
i was having trouble before with hot reload and had to add:  
```
output.publicPath: '/'
```
otherwise it was trying to load chunks from my extensions and looping an error


but instant fix switching between your branch and main...

**Review by okwme — APPROVED** [review]

**Pull request merged**